src/libdrakvuf/private.h

```diff
@@ -270,7 +272,7 @@ struct memaccess
     vmi_mem_access_t access;
 };
 
-struct wrapper
```

Comment:
I would prefer to keep the existing name, it's an unrelated stylistic change to what the PR wants to accomplish

Reply:
I'm with @kscieslinski on this, name `wrapper` provides no information about inside.

Reply:
I would also like to insist on changing the `wrapper` name to something more meaningful. However I'm ok to create a separate pull request for this change and I'm fine with different name, if anyone got better suggestions.